### PR TITLE
tests/extmod: Improve ubinascii.c test coverage.

### PR DESCRIPTION
**Pull request by flowergrass**

### CODE REVIEW

File: tests/extmod/ubinascii_a2b_base64.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 print(binascii.a2b_base64(b'AAECAwQFBgc='))
 print(binascii.a2b_base64(b'CAkKCwwNDg8='))
 print(binascii.a2b_base64(b'f4D/'))
+print(binascii.a2b_base64(b'f4D+'))
```
Comment (dpgeorge) [inline tests/extmod/ubinascii_a2b_base64.py]: Please add a comment for what this is testing.

File: tests/extmod/ubinascii_b2a_base64.py [hunk 2]
```diff
@@ -15,3 +15,4 @@
 print(binascii.b2a_base64(b'\x08\x09\x0a\x0b\x0c\x0d\x0e\x0f'))
 print(binascii.b2a_base64(b'\x7f\x80\xff'))
 print(binascii.b2a_base64(b'1234ABCDabcd'))
+print(binascii.b2a_base64(b'\x00\x00>'))
```
Comment (dpgeorge) [inline tests/extmod/ubinascii_b2a_base64.py]: Please add a comment for what this is testing.

File: tests/extmod/ubinascii_micropython.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+try:
+    import ubinascii as binascii
+except ImportError:
+    import binascii
+
+# two arguments supported in uPy but not CPython
+a = binascii.hexlify(b'123',b'123')
```
Comment (dpgeorge) [inline tests/extmod/ubinascii_micropython.py]: For the second arg, please use ":".

Comment (pfalcon) [inline tests/extmod/ubinascii_micropython.py]: And please follow PEP8 (here space is missing after comma between arguments).

**Comment by dpgeorge** [comment]
Thanks, merged in c15ebf7c8c797af0dc7e601dbb70dbe3777d8ee6.

**Pull request closed without merging**